test/integration/org/org_test.go

```diff
@@ -44,6 +49,30 @@ func getResultFieldStrSlice(rs []gjson.Result, field string) []string {
 	return s
 }
 
+func checkAPIEnabled(t *testing.T, projectID, api string) (string, error) {
```

Comment:
Rather than duplicating code, we could centralize this in a testutils package and reference it here.

Reply:
moved to `testutils` package